File: server/src/main/java/de/litexo/model/OpenttdServerConfig.java
package de.litexo.model;

import lombok.Data;

import java.util.ArrayList;
import java.util.List;

@Data
public class OpenttdServerConfig {
    private String path;
    private int autoSaveMinutes = 5;

    // sha256 hash of the admin password
    private String passwordSha256Hash;

    List<OpenttdServer> servers = new ArrayList<>();
}

File: server/ui/src/app/api/models/openttd-server-config.ts
/* tslint:disable */
/* eslint-disable */
import { OpenttdServer } from './openttd-server';
export interface OpenttdServerConfig {
  autoSaveMinutes?: number;
  passwordSha256Hash?: string;
  path?: string;
  servers?: Array<OpenttdServer>;
}
